components/molecules/table/components/pagination.js

```diff
@@ -8,28 +8,74 @@ import { array, func, number } from 'prop-types'
 
 // UI
 import { Pagination } from '../../../'
+import { Row } from '../../../atoms/'
```

Comment:
@shahabsadeghi Please import directly now this will help with the V2 conflicts

e.g - example only please check actual path - but should end with file
import { Row } from '../../../atoms/grid/components/row'